lib/jira/util/index.js

```diff
@@ -1,6 +1,19 @@
-const { markdown } = require('markdown');
-
 module.exports = function (jiraClient) {
+  const checkForReferenceText = (text) => {
+    const splitTextByNewLine = text.split('\n');
+
+    return splitTextByNewLine
+      // reference text links should have 2 parts only
+      .filter((newLine) => newLine.split(' ').length === 2)
+      .filter((text) => {
+        const hasSquareBrackets = text.charAt(0) === '[' && text.includes(']:');
```

Comment:
maybe combine all the filters into a function `containsReferenceLink()` so that it's clearer what we filter for? We could do something like this, then:

```
return splitTextByNewLine
      .filter((line) => hasReferenceLink(line))
```